docs/.vale/thothtech/InclusionCultural.yml

```diff
@@ -0,0 +1,16 @@
+---
+# Warning: thothtech.InclusionCultural
+#
+# Suggests alternatives for words that are culturally inappropriate.
+#
+# For a list of all options, see https://errata-ai.gitbook.io/vale/getting-started/styles
+extends: substitution
+message: 'Use inclusive language. Consider "%s" instead of "%s".'
+link: TBA
+level: warning
+ignorecase: true
+swap:
+  blacklist(?:ed|ing|s)?: denylist
+  whitelist(?:ed|ing|s)?: allowlist
+  master: primary, main
+  slave: secondary
```

Comment:
**Suggestion:** potential to add a few more words. Hung is one off the top of my head at the moment.

Reply:
demilitarized zone (DMZ) to perimeter network 
Hung to stops responding

Reply:
demilitarized zone (DMZ) to perimeter network 
Hung to stops responding

Reply:
> demilitarized zone (DMZ) to perimeter network

Yep, it is a good idea. I can add that.

> Hung to stops responding

This might be hard to automate though. The word 'hung' can be used in a totally different context. For example, 'The picture was hung on the wall.'. Are you ok if I leave this one out for now?